smart-on-fhir/client-node
# Fix typescript definitions

Since the project is written in typescript it makes more sense to use --declaration to generate types for all the code in src/. 

Testing instructions:
Install the package into another node project that uses typescript locally: `npm install ../client-node/`; 

Statements like:
`import {completeAuth} from 'smart-client';` should now compile.
